React/Views/RCTRefreshControl.m

```diff
@@ -31,6 +32,18 @@ - (instancetype)init
 - (void)layoutSubviews
 {
   [super layoutSubviews];
+  
+  // UIRefreshControl is managed so avoid changing it directly
+  // This method may break in future versions of iOS (subview structure may change)
+  // And self.transform won't shift title text
+  if(_isInitialRender) {
```

Comment:
Great suggestion, @mmmulani! I actually forgot about that I could inspect the UI layers.
I'm thinking of this, which seems to work:
```
NSString *subviewClass = NSStringFromClass([subview class]);
if([subviewClass isEqualToString:@"_UIRefreshControlModernContentView"]) { ... }
```
I will look into adding a test instead of doing this runtime this week.